iina/AppData.swift

```diff
@@ -88,14 +88,20 @@ struct Constants {
 }
 
 extension Notification.Name {
-  static let iinaMainWindowChanged = Notification.Name("IINAMainWindowChanged")
-  static let iinaPlaylistChanged = Notification.Name("IINAPlaylistChanged")
-  static let iinaTracklistChanged = Notification.Name("IINATracklistChanged")
-  static let iinaMediaTitleChanged = Notification.Name("IINAMediaTitleChanged")
-  static let iinaVFChanged = Notification.Name("IINAVfChanged")
-  static let iinaAFChanged = Notification.Name("IINAAfChanged")
-  static let iinaKeyBindingInputChanged = Notification.Name("IINAkeyBindingInputChanged")
-  static let iinaFileLoaded = Notification.Name("IINAFileLoaded")
-  static let iinaHistoryUpdated = Notification.Name("IINAHistoryUpdated")
-  static let iinaLegacyFullScreen = Notification.Name("IINALegacyFullScreen")
+  static let MainWindowChanged = Notification.Name("IINAMainWindowChanged")
```

Comment:
Seems that you forgot to change this line. Also please use camel case for static member name.